kinode/src/sqlite.rs

```diff
@@ -513,6 +514,12 @@ async fn check_caps(
             state
                 .remove_db(request.package_id.clone(), request.db.clone())
                 .await;
+
+            let _ = fs::remove_dir_all(format!(
+                "{}/{}/{}",
+                state.sqlite_path, request.package_id, request.db
+            ));
```

Comment:
We should never swallow `Result`s like this. It makes debugging really hard if an error ever starts being emitted. Either `unwrap()` it or -- at a minimum -- log it